### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by tmshn**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

File: task/git-cli/0.1/README.md [hunk 1]
```diff
@@ -54,4 +54,4 @@ this task in the pipeline.
 
 * If `two factor authentication` is enabled in your git account, then
 `access-token` for the respective git platform must be provided
-in the `password` field while adding secrets.
\ No newline at end of file
+in the `password` field while adding secrets.
```
Comment (vinamra28) [inline task/git-cli/0.1/README.md]: This line should not change. Could you please look at it.

**Review by vinamra28 — CHANGES_REQUESTED** [review]
Thank you for the PR :+1: 
/ok-to-test

**Review by Divyansh42 — APPROVED** [review]
/lgtm

**Review by vinamra28 — APPROVED** [review]
/lgtm

**Review by piyush-garg — APPROVED** [review]
/lgtm

**Review by vdemeester — APPROVED** [review]
/meow

**Pull request merged**